### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by sp4ce-cowboy**
Update PPP, partially completed.

### CODE REVIEW

**Review by lipwei1808 — DISMISSED** [review]
lgtm

File: docs/team/sp4ce-cowboy.md [hunk 1]
```diff
@@ -49,11 +143,12 @@ enhancements that I've implemented or proposed to implement for UniCa$h.
...
     * Added documentation for `ClearTransactions` [to be added]()
 * Developer Guide:
-    * Migrated details of the Developer Guide from an external source and consolidated them in GitHub.
+    * Migrated details of the Developer Guide from an external source (Google Docs) 
+  and consolidated them in GitHub.
 
     
 #### Project Management
-* Managed release v1.2 on GitHub - Can be found [here](https://github.com/AY2324S1-CS2103-T16-3/tp/releases/tag/v1.2). 
+* Managed release v1.3a on GitHub - Can be found [here](https://github.com/AY2324S1-CS2103-T16-3/tp/releases/tag/v1.2).
```
Comment (elhy1999) [inline docs/team/sp4ce-cowboy.md:151]: Is this link wrong since it's supposed to be for v1.3a instead of v1.2?

**Review by elhy1999 — CHANGES_REQUESTED** [review]

File: docs/team/sp4ce-cowboy.md (hunk 1 above)
Comment (sp4ce-cowboy) [reply docs/team/sp4ce-cowboy.md]: Ok I've fixed it

**Review by elhy1999 — APPROVED** [review]
LGTM

**Pull request merged**